packages/react-components/src/templates/__tests__/TeamProfileHeader.test.tsx

```diff
@@ -92,7 +92,7 @@ it('renders tabs', () => {
   const { getAllByRole } = render(<TeamProfileHeader {...boilerplateProps} />);
   expect(getAllByRole('link').map(({ textContent }) => textContent)).toEqual([
     'About',
-    'Outputs',
+    'Team Outputs',
```

Comment:
ewwwww brittle tests
if you don't want these needing to change again, change to `expect.stringMatching(/output/i)` etc. :sweat_smile:

Reply:
:fire: